apps/sharing/__init__.py

```diff
@@ -13,62 +13,36 @@ def count_term(count):
         """Render this service's share count with the right term."""
         return ngettext('{0} post', '{0} posts', count).format(count)
 
-
-class DELICIOUS(ServiceBase):
-    """see: http://delicious.com/help/savebuttons"""
-    shortname = 'delicious'
-    label = _(u'Add to Delicious')
-    url = (u'http://delicious.com/save?url={url}&title={title}'
-           u'&notes={description}')
-
-
-class DIGG(ServiceBase):
```

Comment:
Now that the code's syntax is correct, we can see that at least one test was using those classes, and need to be fixed: https://travis-ci.org/mozilla/olympia/jobs/62702292#L462